src/App.js

```diff
@@ -6,21 +6,22 @@ import usePersistedState from "./hooks/usePersistedState";
 import Menu from "./pages/Menu";
 import Order from "./pages/Order.js";
 import Bill from "./pages/BillPage.js";
+import CheckIn from "./components/CheckIn.js";
 import ConfirmServiceText from "./pages/ConfirmServiceText.js";
 import ConfirmOrderText from "./pages/ConfirmOrderText";
 import ConfirmBillText from "./pages/ConfirmBillText";
-
 import Navigation from "./components/Navigation";
 import Div100vh from "react-div-100vh";
 
 function App() {
   const [menu, setMenu] = useState([]);
   const [order, setOrder] = usePersistedState("order", []);
+  const [tableId, setTableId] = usePersistedState("table", 0);
 
   let history = useHistory();
 
   useEffect(() => {
-    const url = "/api/menu.json";
+    const url = `/api/menu.json`;
```

Comment:
why did you change it to ``? you are using "" everywhere else. :)

Reply:
Actually I tried something else there and just didn't change it back 😄  haha